### DISCUSSION TITLE
Customise Weapons?

### DISCUSSION THREAD

**Discussion by CursedSFMS** [Hydraxous/UltraFunGuns#45] [Suggestions]
Don't know if this has already been asked for but could you please add the option to customise the looks of the weapons? They look fine is all just want the option to have a gold Desert Eagle or Metallic Grey Sniper Rifle just so there's more options for looks is all if not its fine

**Comment by Hydraxous** [comment]
There is some framework for this being implemented. Custom colors like the vanilla game will not be implemented. However, the wip feature allows you to swap the textures to custom ones. If I can get it working in a desirable way, it will be added.
